docs/release-notes/index.md

```diff
@@ -60,11 +60,11 @@ For product updates that are in progress or planned [visit the Codacy public roa
 
 ## Codacy Self-hosted release notes {: id="self-hosted"}
```

Comment:
:stop_sign: There was an issue while publishing the docs for version 6.0.0, let's wait until I make it work before merging this pull request.